contracts/upgradeable_contracts/amb_erc677_to_erc677/BasicAMBErc677ToErc677.sol

```diff
@@ -67,6 +67,15 @@ contract BasicAMBErc677ToErc677 is
         return isInitialized();
     }
 
+    function getReceiver(address _from, bytes _data) internal view returns (address recipient) {
+        recipient = _from;
+        if (_data.length == 20) {
+            recipient = Bytes.bytesToAddress(_data);
+        }
```

Comment:
Do you think it makes sense to revert in a case when data exists but it is not address?